packages/cli/src/cliEntry.js

```diff
@@ -70,16 +70,15 @@ function printHelpInformation() {
 }
 
 function printUnknownCommand(cmdName) {
-  logger.error(
-    [
-      cmdName
-        ? `Unrecognized command "${cmdName}".`
-        : "You didn't pass any command.",
-      `Run ${chalk.white(
-        '"react-native --help"'
-      )} to see list of all available commands.`,
-    ].join(' ')
-  );
+  cmdName
+    ? logger.error(
+        [
+          `Run ${chalk.white(
+            '"react-native --help"'
+          )} to see list of all available commands.`,
+        ].join(' ')
```

Comment:
Logger.error will join arguments with a space so no need for that now (array with join is a legacy behavior we changed revently)

Reply:
Removed, thanks for pointing out!